src/gmtmex/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+#
+#
+# Copyright (c) 1991-2020 by the GMT Team (https://www.generic-mapping-tools.org/team.html)
+# See LICENSE.TXT file for copying and redistribution conditions.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; version 3 or any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# Contact info: www.generic-mapping-tools.org
+#-------------------------------------------------------------------------------
+#
+# CMake settings for supplemental package:
+#
+#   SUPPL_NAME:          name of the supplemental package
+#   SUPPL_HEADERS:       header files. Will be installed if BUILD_DEVELOPER is TRUE
+#   SUPPL_PROGS_SRCS:    list of C source codes for supplemental modules
+#   SUPPL_LIB_SRCS:      list of C source codes for supplemental library
+#   SUPPL_EXAMPLE_FILES: README and other example files
+#   SUPPL_EXAMPLE_PROGS: Example scripts
+#
+
+set (SUPPL_NAME gmtmex)
+set (SUPPL_PROGS_SRCS gmtmex.c gmtmex_parser.c)
+set (SUPPL_HEADERS gmtmex.h)
+set (SUPPL_LIB_SRCS ${SUPPL_PROGS_SRCS})
```

Comment:
This may work:
```suggestion
set (SUPPL_HEADERS gmtmex.h)
set (SUPPL_LIB_SRCS ${SUPPL_PROGS_SRCS} gmtmex.c gmtmex_parser.c)
```

Reply:
Agree with header and parser.  But gmtmex.c is like gmt.c and is to become a stand-alone executable.  So perhaps just parser?

Reply:
gmtmex.c is not a module, so can't be in SUPPL_PROGS_SRCS. Perhaps need to define a new cmake variable, and also update src/CMakeLists to be able to build stand-alone programs.

Reply:
Yes, I will leave it off for now since it is different.